nixos/modules/services/desktops/gnome3/gvfs.nix

```diff
@@ -4,6 +4,12 @@
 
 with lib;
 
+let
+
+  cfg = config.services.gnome3.gvfs;
```

Comment:
Maybe we could move it out to `services.gvfs`.

Reply:
For sure, since it's used in xfce, mate, and lxqt.